examples/cdf_apm_simple/default.config.yaml

```diff
@@ -10,10 +10,9 @@ view_WorkOrder_version: '1'
 view_WorkItem_version: '1'
 clientId: ${IDP_CLIENT_ID}
 clientSecret: ${IDP_CLIENT_SECRET}
-tokenUrl: ${IDP_TOKEN_URL}
+tokenUri: ${IDP_TOKEN_URL}
 # Optional: If idP requires providing the scopes
 cdfProjectName: ${CDF_PROJECT}
-scopes:
-  - ${IDP_SCOPES}
+scopes: ${IDP_SCOPES}
```

Comment:
Scopes already comes as a list from `IDP_SCOPES` which caused this to become a list of list.